app/views/list/index.html.slim

```diff
@@ -0,0 +1,44 @@
+html[lang="en"]
```

Comment:
Anything `html` and `body` related should be pulled into `app/views/layouts/application.html.slim`.